horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/RunServiceImpl.java

```diff
@@ -188,9 +189,20 @@ void onNewOrUpdatedSchema(int schemaId) {
    }
 
    void findRunsWithUri(String uri, BiConsumer<Integer, Integer> consumer) {
-      ScrollableResults results = Util.scroll(em.createNativeQuery(FIND_RUNS_WITH_URI).setParameter(1, uri));
+      ScrollableResults<RunFromUri> results =
+             em.createNativeQuery(FIND_RUNS_WITH_URI).setParameter(1, uri)
```

Comment:
we should define an optional `tupleTransformer` param for `Util.scroll` and pass in the tuple lambda to remove duplicated code